pkg/transformers/labelsandannotationsconfig.go

```diff
@@ -85,7 +85,7 @@ var defaultLabelsPathConfigs = []PathConfig{
 	{
 		GroupVersionKind:   &schema.GroupVersionKind{Group: "batch", Kind: "Job"},
 		Path:               []string{"spec", "selector", "matchLabels"},
-		CreateIfNotPresent: true,
+		CreateIfNotPresent: false,
```

Comment:
Can you also do that for CronJob？ Thank you

Reply:
Sure, I'll play with it some. #116 seems to actually be spec.jobTemplate.spec.template.metadata. I haven't used a CronJob before, but I'll play around and make sure they get accepted by K8s.